src/microbe_stage/Microbe.cs

```diff
@@ -473,13 +473,38 @@ public override void _Ready()
             if (ColonyChildren != null)
             {
                 foreach (var child in ColonyChildren)
+                {
                     AddChild(child);
+
+                    // Re-adds the colony members to collision exception list of each microbe in colony
+                    // Then with the master collision exception list
+                    foreach (var member in ColonyChildren)
+                    {
+                        if (member != child)
+                        {
+                            member.AddCollisionExceptionWith(child);
+                            child.AddCollisionExceptionWith(member);
+                        }
+                    }
+
+                    AddCollisionExceptionWith(child);
+                    child.AddCollisionExceptionWith(this);
+                }
             }
 
             // Need to re-attach our organelles
             foreach (var organelle in organelles)
                 OrganelleParent.AddChild(organelle);
 
+            // Colony children shapes need reparenting to their master
+            if (!IsPlayerMicrobe && Colony != null)
+            {
+                ReParentShapes(this, Vector3.Zero);
+                ReParentShapes(Colony.Master, (GlobalTransform.origin - Colony.Master.GlobalTransform.origin).Rotated(
```

Comment:
It is why wouldnt it be?

Reply:
I was thinking of https://github.com/Revolutionary-Games/Thrive/pull/2461 Now I remember that wasn't merged yet.
But do you think that this math needs changes when that PR is merged?

Reply:
By initial attaching you mean when the colony members are reatached when loaded?

Reply:
Yeah, I mean the attach when loading a save and when attaching a new member to a colony. These two things seem to have quite a few of lines in common involving some tricky math, so I'd really like to put that to a method that both can call to not have that code in two places.

Reply:
Btw,it wont work if i only try to reparent it to the master again,it only works if i reparent the shpes to the microbe then to master again sooo,idk engine bug again?

Reply:
> This only is happeneing after loading so i dont see the need for a method because its only used once and if you mean the initial attaching in game it doesnt need to reparents.

I'm talking about the math like `(GlobalTransform.origin - Colony.Master.GlobalTransform` that seems to be in two places. That's the thing I'd like to have in a separate method.

> Btw,it wont work if i only try to reparent it to the master again,it only works if i reparent the shpes to the microbe then to master again sooo,idk engine bug again?

Maybe? Please add a comment about that why that code is here, then.